benchmarks/TFLite/CMakeLists.txt

```diff
@@ -104,6 +104,19 @@ set(MOBILEBERT_FP16_MODULE
     "1x384xi32,1x384xi32,1x384xi32"
 )
 
+set(MOBILENET_V1_MODULE
+  NAME
+    "MobileNetV1"
+  TAGS
+    "fp32,imagenet"
+  SOURCE
+    "https://storage.googleapis.com/iree-model-artifacts/mobilenet_v1_224_1.0_float.tflite"
```

Comment:
Is there a model source for this? Like other "Mirror of" comments.

Reply:
It is used directly as is in https://github.com/google/iree-samples/tree/903fadef81365faa91601e32c481ad2cdbc21206/tflitehub/mobilenet_v1_test.py#L8

Reply:
Marie is not available for the next 2.5 weeks. I will add a TODO here to track it.

Reply:
I think Marie has been adding them to our tfhub. This is the one I've been using for mobilenet V1: https://tfhub.dev/iree/lite-model/mobilenet_v1_100_224/fp32/1. Hopefully they are the same.

I'm just curious... why are we mirroring them instead of using the tfhub one directly?

Reply:
IIUC we can control the `iree-model-artifacts` GCS bucket better. the tfhub GCS is not solely maintained by IREE so there's a chance the artifacts get removed by other admins, as well as pulling from the tensorflow github URL could hit timeout and fail to fetch.